apps/operators/external-dns.yaml

```diff
@@ -1,5 +1,16 @@
 ---
 component: external-dns
 sources:
-  - ref: understack
-    path: 'operators/external-dns'
+  - repoURL: ghcr.io/rackerlabs/charts
+    path: .
+    chart: external-dns-rackspace
+    targetRevision: 0.0.1
+    includeCRDs: true
```

Comment:
This line isn't valid. https://argo-cd.readthedocs.io/en/stable/user-guide/helm/